code/style.css

```diff
@@ -1,4 +1,385 @@
+  overflow: hidden;
+  /* max-height: 2em; */
+  display: -webkit-box;
+  -webkit-line-clamp:2; /* number of lines to show */
+  -webkit-box-orient: vertical;
+}
+
+.small-text{
+  color: #383C4B;
+  font-weight: 200;
+  margin: 0.5rem 0rem 2rem 0rem;
+  text-align: center;
+}
+.more-text{
+  color: #383C4B;
+  font-weight: 200;
+  margin: 0.5rem 0rem 2rem 0rem;
+}
+hr{
+  border: 2px solid #84ACB6;
+  width: 70%;
+  border-radius: 50px;
+  align-self: center;
+  max-width: 500px;
+  animation: fadeIn 10s;
+  opacity: 1;
+}
+
+
+@keyframes fadeIn {
+  from {
+  	opacity: 0;
+  }
+  to {
+ 	opacity: 1;
+  }
+}
+
+
+/*mobile*/
+@media (min-width:0px) and (max-width:767px){
+  nav{
+    display: none;
+  }
+.more-text{
+  display: none;
+}
+.logo-big{
+  display: none;
+}
+}
+
+/*tablet*/
+ @media (min-width: 768px) and (max-width: 991px) {
+
+  .logo{
+    display: none;
+  }
+  header{
+    /* display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-areas: "head1 head2 head3 head4 head5";
+    grid-template-rows: auto;
+    grid-gap: 1em;  */
+    
+   
+  }
+nav{
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-areas: "head1 head2 head3 head4 head5";
+  grid-template-rows: auto;
+  grid-gap: 1em;
+  align-items: center; 
+  justify-items: center;
+
+}
+  .logo{
+  grid-area: head3;
+  }
+
+.nav1{
+  grid-area: head1;
+  color: #383C4B;
+  text-decoration: none;
+}
+.nav2{
+  grid-area: head2;
+  color: #383C4B;
+  text-decoration: none;
+}
+.nav3{
+  grid-area: head4;
+  color: #383C4B;
+  text-decoration: none;
+}
+.nav4{
+  grid-area: head5;
+  color: #383C4B;
+  text-decoration: none;
+}
+  .meny{
+    display: none;
+  }
+  h1{
+    display: none;
+  }
+  .wrapper{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap:30px;
+  }
+  .big-news{
+  grid-column-start: span 2;
+  width: 100%;
+  justify-self: stretch;
+  justify-self: center;
+  max-width: 900px;
+  flex-direction: row;
+  align-items:flex-start;
+  padding: 1rem;
+  }   
+  hr{
+  grid-column-start: span 2;
+  justify-self: center;
+  }
+  .a-nav{
+    margin-right: 20px;
+  }
+  .big-news-image{
+   max-width: 50%;
+   width: 50%;
+   align-self: flex-start;
+   padding: 2rem;
+   border-radius: 20px;
+   align-self: center;
+  }
+  h2{    
+    max-width: 400px;
+    margin: 40px 40px 40px 0px;
+    text-align: left;
+    font-size: 2em;
+  }
+  h3{
+    font-size: 1.2em;
+  }
+  .more-text{
+
+    position: relative;
+
+  }
+  .small-text{
+    display: none;
+  }
+
+  .text-container{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+  }
+ }
+
+
+
+
+/*desktop*/
+@media (min-width: 992px) {
+  .meny{
+    display: none;
+  }
+  .logo-big{
+    display: none;
+  }
+
+  .small-text{
+    display: none;
+  }
+
+  .wrapper{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap:30px;
+    margin: 0 30px;
+  }
+  h2{
+    font-size: 3em;
+  }
+
+  h3{
+    font-size: 1.1em;
```

Comment:
nice to change the font size when changing from mobile/tablet/desktop version. will do these small changes to to make it look professional :)